src/mudata/_core/file_backing.py

```diff
@@ -18,7 +18,7 @@ def __init__(
         self._children = WeakSet()
         if filename is not None:
             filename = Path(filename)
-        super().__init__(None, filename, filemode)
+        super().__init__(ad.AnnData(), filename, filemode)
```

Comment:
This is the one I'm least sure of tbh.  Seems like `None` and `ad.AnnData` are just as good (bad) here but 🤷